tools/ci_build/github/azure-pipelines/web-ci-pipeline.yml

```diff
@@ -1,3 +1,31 @@
+trigger:
+  branches:
+    include:
+    - main
+    - rel-*
+  paths:
+    include:
+    - 'js/web'
+    - 'js/node'
+    - 'onnxruntime'
+    exclude:
+    - 'onnxruntime/core/providers/tensorrt'
+    - 'onnxruntime/core/providers/cuda'
+    - 'onnxruntime/core/providers/rocm'
+pr:
+  branches:
+    include:
+    - main
+    - rel-*
+  paths:
+    include:
```

Comment:
This is too aggresive. a lot of changes should also be caught by web CI:
submodules
cmake
include
tools